src/Cohesity.Powershell/Scripts/VirtualIP/Remove-CohesityVirtualIP.ps1

```diff
@@ -0,0 +1,74 @@
+function Remove-CohesityVirtualIP {
+    <#
+        .SYNOPSIS
+        Remove the virtual IP(s).
+        .DESCRIPTION
+        The Remove-CohesityVirtualIP function is used to remove virtual IP(s).
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Remove-CohesityVirtualIP -VlanName "intf_group2.0" -VirtualIPs "10.3.144.14", "10.3.144.15"
```

Comment:
Make sure you give more generic IPs 1.2.3.4

Reply:
Fixed